src/_guides/language/analysis-options.md

```diff
@@ -239,7 +239,34 @@ include: package:pedantic/analysis_options.yaml
   or [disable individual rules][].
 {{site.alert.end}}
 
-[disable individual rules]: #disabling-individual-rules
+### Enabling the set of rules corresponding to the Effective Dart guide: effective_dart {#enabling-effective-dart-rules}
```

Comment:
Let's make this heading shorter. Maybe:

Enabling Effective Dart rules: effective_dart